internal/mq/msgstream/mq_msgstream.go

```diff
@@ -485,6 +485,10 @@ func (ms *mqMsgStream) receiveMsg(consumer mqwrapper.Consumer) {
 			if !ok {
 				return
 			}
+			if msg == nil {
```

Comment:
chan should block if no error?
when shall msg be nil?

Reply:
the block is another strategy